client/configuration_variable_test.go

```diff
@@ -82,6 +96,22 @@ var _ = Describe("Configuration Variable", func() {
 		It("Should return created configuration variable", func() {
 			Expect(createdConfigurationVariable).To(Equal(mockConfigurationVariable))
 		})
+
+		DescribeTable("Create with different schema format", func(schemaFormat Format) {
+			var mockWithFormat = ConfigurationVariable{}
+			copier.Copy(&mockWithFormat, &mockConfigurationVariable)
+			mockWithFormat.Schema.Format = schemaFormat
+			requestWithFormat := GetExpectedRequest(mockWithFormat)
+
+			DoCreateRequest(mockWithFormat, requestWithFormat)
+
```

Comment:
was not easy to follow here, maybe take `requestWithFormat` inside `DoCreateRequest` because they both use `mockWithFormat`?

Reply:
changed in https://github.com/env0/terraform-provider-env0/pull/196/commits/67d139d9a15e3f7ffdc90167a5734850bb6de2a6

LMK WDYT